server/girder_plugin/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+from pathlib import Path
+from typing import Optional, Tuple
+
+from girder import events, plugin
+from girder.models.setting import Setting
+
+from .client_webroot import ClientWebroot
+
+class GirderPlugin(plugin.GirderPlugin):
+    def load(self, info):
+        # Relocate Girder
+        info["serverRoot"], info["serverRoot"].girder = (
+            ClientWebroot(),
+            info["serverRoot"],
+        )
+        info["serverRoot"].api = info["serverRoot"].girder.api
+
+        # Create dependency on worker
+        # plugin.getPlugin('worker').load(info)
+        # Setting().set(
+        #     'worker.api_url',
+        #     os.environ.get('WORKER_API_URL', 'http://girder:8080/api/v1'),
+        # )
+        # Setting().set(
+        #     'worker.broker',
+        #     os.environ.get('CELERY_BROKER_URL', 'amqp://guest:guest@rabbit/'),
+        # )
+        # Setting().set(
+        #     'worker.backend',
+        #     os.environ.get('CELERY_BROKER_URL', 'amqp://guest:guest@rabbit/'),
+        # )
```

Comment:
Worker's role in this project is somewhat unclear to me.  I guess it's just for opaque large image background jobs?  I can probably remove this comment block.

Reply:
We don't need a remote worker for this project; all of our images should be natively readable by large image.